### ISSUE TITLE
Prevent Defaults/Optional param docs

### ISSUE THREAD

**Issue by brettz9** [gajus/eslint-plugin-jsdoc#477]
Since ES6 allows for default params, projects might not wish to track this information redundantly in jsdoc, so this rule would report any default docs like:

```js
/**
 * @param {SomeType} [aParam='theDefault'] 
*/
function quux (aParam = 'theDefault') {
}
```

With an option it could also disallow optional syntax:

```js
/**
 * @param {SomeType} [aParam] 
*/
function quux (aParam = 'theDefault') {
}
```

...for projects that wished all optional prams to be given a formal ES6 default.

(This rule might not be for everyone, including for the reason that default prams don't providing defaulting on non-`undefined` values such as `null` or other falsey values.)

I don't think a corresponding `@property` rule would be needed since there is no special ES6 for defaulting on object properties (just the usual `||`).
